internal/applicationsnapshot/report_test.go

```diff
@@ -150,28 +150,27 @@ components:
     containerImage: quay.io/caf/spam@sha256:123…
     violations: []
     warnings: null
+    successes: null
     success: true
-    successCount: 5
   - name: bacon
     containerImage: quay.io/caf/bacon@sha256:234…
     violations: []
     warnings: null
+    successes: null
     success: true
-    successCount: 5
   - name: eggs
     containerImage: quay.io/caf/eggs@sha256:345…
     violations: []
     warnings: null
+    successes: null
     success: true
-    successCount: 5
 `
 
 	var components []Component
 	for _, component := range snapshot.Components {
 		c := Component{
-			Violations:   []output.Result{},
-			Success:      true,
-			SuccessCount: 5,
```

Comment:
Where did the 5 come from? :thinking:

Reply:
This test kinda sucks, I'm going to refactor it